src/main.go

```diff
@@ -323,6 +540,26 @@ OPTIONS:
 	app.EnableBashCompletion = true
 	app.HideVersion = true
 	app.Commands = []*cli.Command{
+		&cli.Command{
+			Name:      "admin",
+			Usage:     "Admin commands for managing lambdas",
+			UsageText: "ol admin <cmd>",
+			Subcommands: []*cli.Command{
```

Comment:
let's put admin commands to a new file (package?), like we do for with BenchCommands and WorkerCommands